### DISCUSSION TITLE
Seq running in AWS ECS - "Termination signal received"

### DISCUSSION THREAD

**Discussion by rj-au** [datalust/seq-tickets#2153] [Help] [answered]
The Seq container (AWS ECS) reported receiving a "Termination signal".
We received 503/504 errors from Seq running in AWS ECS around that time.
Is there anything we can do to reduce the likelihood of this happening again?
And what would be the cause(s)?

These are the logs:
![image](https://github.com/datalust/seq-tickets/assets/106497642/3c9a45bd-d3c6-4ec0-a4c0-f9a6841a4f4c)

CPU/Memory (4 weeks):
![image](https://github.com/datalust/seq-tickets/assets/106497642/c3d6f530-f6ab-48cf-900c-4e063675802f)

1. Same thing happened on the 6th March
2. Terminated today (3rd April)

CPU/Memory (12 hours):
![image](https://github.com/datalust/seq-tickets/assets/106497642/9f32f91e-5294-4315-8255-9249603d66cd)

System settings:
![image](https://github.com/datalust/seq-tickets/assets/106497642/86cc2ad9-a872-4d01-b790-283da84d2124)

Retention rules:
![image](https://github.com/datalust/seq-tickets/assets/106497642/193c31bc-0189-47e1-814a-1a085f36d7fa)

**Comment by nblumhardt** [comment]
Hi @richard-stafflink,

We don't have any specifics on how to configure ECS for this, but a custom AWS maintenance window should help narrow down the time of day that container recycling is performed. It'd be great to hear whether you're able to use this.

Hope this helps,
Nick

**Reply by rj-au** [reply]
Hi @nblumhardt,

We haven't configured any container restarts or maintenance windows, so I'm not sure how the termination signal came about.

**Reply by nblumhardt** [reply, marked as answer]
Hi @richard-stafflink; AWS itself performs restarts when rebalancing load across hosts, or when the host the container is on requires maintenance.

**Reply by rj-au** [reply]
That makes sense, thank you.
https://repost.aws/knowledge-center/ecs-task-stopped

**Comment by rj-au** [comment]
Restart occurred due to ECS event: https://repost.aws/knowledge-center/ecs-task-stopped
